lang/main.json

```diff
@@ -422,5 +422,12 @@
         "seconds": "__count__s",
         "speakerStats": "Speaker Stats",
         "speakerTime": "Speaker Time"
+    },
+    "deviceSelection": {
+        "deviceSettings": "Device settings",
+        "noOtherDevices": "No other devices available",
+        "noThanks": "No thanks",
```

Comment:
@saghul+++++ I haven't noticed that and it's actually my fault that I haven't named the buttons correctly on the mockups. It should be "Cancel". Lenny can you use cancel instead? Thanks!